projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -225,6 +231,10 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
   }
 
   private resolveContent() {
+    if (this.content === null || this.content === undefined) {
```

Comment:
How do we get it here if we don't create an instance?

Reply:
Sorry, you're right. I missed that. It will be removed with the next push.